ci-operator/step-registry/interop/mtr/execute/README.md

```diff
@@ -0,0 +1,164 @@
+sed -i "s#REPLACE_OCP_HOSTNAME#${OCP_HOSTNAME}#" $CONFIG_FILE
+sed -i "s#REPLACE_OCP_SECURE_HOSTNAME#${OCP_SECURE_HOSTNAME}#" $CONFIG_FILE
+sed -i "s#REPLACE_FTP_HOST#${FTP_HOST}#" $CONFIG_FILE
+sed -i "s#REPLACE_FTP_USERNAME#${FTP_USERNAME}#" $CONFIG_FILE
+sed -i "s#REPLACE_FTP_PASSWORD#${FTP_PASSWORD}#" $CONFIG_FILE
+sed -i "s#REPLACE_EXECUTOR#${SELENIUM_EXECUTOR}#" $CONFIG_FILE
+sed -i "s#REPLACE_MTR_PASSWORD#${MTR_PASSWORD}#" $CONFIG_FILE
+sed -i "s#REPLACE_MTR_USER#${MTR_USERNAME}#" $CONFIG_FILE
+sed -i "s#REPLACE_MTR_VERSION#${MTR_VERSION}#" $CONFIG_FILE
+```
+
+### Install MTR Tests
+
+The following code snippet uses `pip` to install the MTR tests from the `/tmp/integration_tests` directory within the container. These tests come from the the [windup/windup_integration_test](https://github.com/windup/windup_integration_test.git) repository maintained by MTR product QE. These tests must be installed in this script rather than when the container image is built because OpenShift runs these containers using a user that ends up not having access to [modify the configuration file](#create-mtr-test-configuration-file) needed to execute these tests. Because that configuration file changes with every run, we have to modify it *then* install the tests.
+
+```bash
+echo "Installing integration tests"
+pip install -e /tmp/integration_tests
+```
+
+### Start the local FTP Server
+
+Because these tests require an FTP server and the one used previously is behind our firewall, this image contains a script that will start a local FTP server that holds the `.war` needed to execute the tests. The following command will start the server in the background.
+
+```bash
+echo "Starting the local FTP Server"
+python /tmp/ftp_server.py &
+```
+
+### Execute Tests
+
+The following code snippet uses `pytest` to execute the Interop MTR tests. The XUnit/JUnit results are then published to the `${SHARED_DIR}/xunit_output.xml` file. This file is to be used in the [interop-mtr-report](../report/README.md) step of this scenario.
+
+```bash
+echo "Executing PyTest..."
+pytest /tmp/integration_tests/mta/tests/operator/test_operator_test_cases.py -vv --reruns 4 --reruns-delay 10 --junitxml=${SHARED_DIR}/xunit_output.xml
+```
+
+### Stop the local FTP Server
+
+The following line of code will stop the local FTP server that was started earlier in the script. If the process is left running, the execute pod will not complete and OpenShift CI will stop the pod after 2 hours, failing the execution.
+
+```bash
+pkill -f ftp_server.py
+```
+
+## Container Used
+
+The container used in this step is named `mtr-runner` in the [configuration file](../../../../windup/windup-windup_integration_test-mtr.yaml). This container created from a custom image located in the [windup/windup_integration_test](https://github.com/windup/windup_integration_test.git) repository within in the `dockerfiles/interop` directory. The code snippet below is the Dockerfile found in that repository.
+
```

Comment:
Wouldn't a link to the file be better (so if changed, this doc will not need to be updated)?

Reply:
See [90600e36f457961ad427db2012821f66007e76db](https://github.com/openshift/release/pull/34094/commits/90600e36f457961ad427db2012821f66007e76db)